pom.xml

```diff
@@ -827,6 +831,51 @@
                 <apt.phase>compile</apt.phase>
             </properties>
         </profile>
+        <profile>
+            <id>java8</id>
+            <activation>
+                <jdk>1.8</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <version>3.5.0</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>properties</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
```

Comment:
What would be the purpose of this one? Does it deserve a comment?

Reply:
This is needed to set the value of the project property `${org.mule.apache:xerces2-xsd11:jar:}`, needed for `maven-surefire-plugin`.